do.sh

```diff
@@ -337,12 +314,51 @@ function run_test() {
 
     # Collecting stats only for 3 workers to avoid bloating the report.
     resource_usage_logs=$(find ${out_dir}/logs -name process-stats.json \
-                            | grep ovn-scale | head -3 || true)
+                            | grep ovn-scale | head -3)
     python3 ${topdir}/utils/process-stats.py \
         resource-usage-report-worker.html ${resource_usage_logs}
+    popd
+}
 
-    deactivate
```

Comment:
We should, probably, still deactivate somewhere...

Reply:
Oops, sorry about that.  Fixed now.